src/test/java/com/cloudant/tests/CloudantClientHelper.java

```diff
@@ -27,13 +27,13 @@
  */
 public abstract class CloudantClientHelper {
 
-    protected static final String COUCH_USERNAME;
-    protected static final String COUCH_PASSWORD;
-    protected static final String COUCH_HOST;
-    protected static final String COUCH_PORT;
-    protected static final String HTTP_PROTOCOL;
-    protected static final URI SERVER_URI;
-    protected static final boolean IGNORE_CLOUDANT_SPECIFIC;
+    public static final String COUCH_USERNAME;
+    public static final String COUCH_PASSWORD;
+    public static final String COUCH_HOST;
+    public static final String COUCH_PORT;
+    public static final String HTTP_PROTOCOL;
+    public static final URI SERVER_URI;
+    public static final String SERVER_URI_WITH_USER_INFO;
```

Comment:
This class confuses me because there are so many different ways of specifying the URL - although I know a lot of this is for historical reasons.

What happens if I supply `test.couch.uri` with a "userinfo subcomponent" - username and password? Will cookies still get used? Your PR description says "Removed user info from CloudantClientHelper SERVER_URL" but it appears that this doesn't happen if you go down this code path.